pkg/tfgen/docs.go

```diff
@@ -1049,6 +1050,17 @@ func parseAttributesReferenceSection(subsection []string, ret *entityDocs) {
 }
 
 func (p *tfMarkdownParser) parseImports(subsection []string) {
+	var token string
+	if p.info != nil && p.info.GetTok() != "" {
+		token = p.info.GetTok().String()
+	}
+	defer func() {
+		contract.Assertf(!strings.Contains(p.ret.Import, "erraform"),
```

Comment:
OK this turns out to be too strict for random and azure-ad downstream checks. Need to reconsider or downgrade to a warning... possibly some data cleanup for us to do on these.

Reply:
Warnings in docsgen are almost completely useless on large providers. If an import section (post processing) still contains the word "erraform", can we just drop the import section completely.

We can also add a warning, but I don't think it will change our behavior.

Reply:
Well, what can we do to change that? I'll add a note to ideation.